packages/wrangler/package.json

```diff
@@ -105,7 +105,7 @@
 		"@miniflare/durable-objects": "2.12.1",
 		"blake3-wasm": "^2.1.5",
 		"chokidar": "^3.5.3",
-		"esbuild": "0.16.3",
+		"esbuild": "0.16.5",
```

Comment:
Was the "external" feature from ESBuild needed for the feature to be complete? Just trying to understand the ESBuild bump and "external" feature interaction with the `--no-bundle` feature  😁

Reply:
Yep, the `external` feature is needed. This PR uses `esbuild` to try and figure out which additional modules should be uploaded with your entrypoint file. Without `packages: external` we'd try and upload your package imports as well as just file imports. This is pretty much only relevant for `node:*` node-compat imports right now, but more might be added in future. 

We _could_ try and figure out which package imports could be safely uploaded, to support runtime importing of `npm` modules, but the right solution for that is bundling.

Reply:
Appreciate the clarification!! 🎆

Reply:
> the right solution for that is bundling.
So like https://www.npmjs.com/package/rollup lol 😆